File: programmers/level1/K번째수.java
package programmers;
import java.io.*;
import java.util.*;
public class K번째수 {
	static 
	class Solution {
	    public int[] solution(int[] array, int[][] commands) {
	        int[] answer = new int[commands.length];
	        for(int idx = 0; idx < commands.length; ++idx) {
	        	int i = commands[idx][0]-1;
	        	int j = commands[idx][1]-1;
	        	int k = commands[idx][2]-1;
	        	int size = j-i+1;
	        	int[] splited = new int[size];
	        	for(int s = i; s <= j; ++s) {
	        		splited[s-i] = array[s];
	        	}
	        	Arrays.sort(splited);
	        	answer[idx] = splited[k];
	        }
	        return answer;
	    }
	}
	public static void main(String[] args) {
		int[] array = {1,5,2,6,3,7,4};
		int[][] commands = {{2,5,3},{4,4,1},{1,7,3}};
		Solution sol = new Solution();
		System.out.println(Arrays.toString(sol.solution(array, commands)));
	}
}

File: programmers/js/level1/k번째수.js
function solution(array, commands) {
    var answer = [];
    for (let i = 0; i < commands.length; ++i) {
        const from = commands[i][0] - 1;
        const to = commands[i][1];
        const k = commands[i][2] - 1;
        let temp = [];
        for (let j = from; j < to; ++j) {
            temp.push(array[j]);
        }
        temp.sort((a, b) => {
            return a - b;
        })
        answer.push(temp[k]);
    }
    return answer;
}

console.log(solution([1, 5, 2, 6, 3, 7, 4], [[2, 5, 3], [4, 4, 1], [1, 7, 3]]));